lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/security/FileBasedAccessControlConfig.java

```diff
@@ -14,33 +14,46 @@
 package io.trino.plugin.base.security;
 
 import io.airlift.configuration.Config;
-import io.airlift.configuration.validation.FileExists;
+import io.airlift.configuration.ConfigDescription;
 import io.airlift.units.Duration;
 import io.airlift.units.MinDuration;
 
 import javax.validation.constraints.NotNull;
 
-import java.io.File;
-
 public class FileBasedAccessControlConfig
 {
     public static final String SECURITY_CONFIG_FILE = "security.config-file";
+    public static final String SECURITY_JSON_POINTER = "security.json-pointer";
     public static final String SECURITY_REFRESH_PERIOD = "security.refresh-period";
 
-    private File configFile;
+    private String configFilePath;
+    private String jsonPointer = "";
     private Duration refreshPeriod;
 
     @NotNull
-    @FileExists
```

Comment:
Can we revert this change ? `FileExists` is an useful annotation which would help us to fail fast if the file doesn't exist instead of failing with a different message

Reply:
filepath might be now local file ot URL (starting with https or http). I think FileExists won't be able to handle this case

Reply:
In this case I'm not sure if it would be a right approach to share the config.

Reply:
What about splitting configs but also modules into separate ones for remote and local file? Right now in FileBasedAccessControlModule we logic for both local and remote/rest. What about having for example two modules: LocalFileBasedAccessControlModule and RemoteFileBasedAccessControlModule? What do you think?

Reply:
Having a different config is a better option, as it allows us to isolate the implementation specific config objects.

Reply:
there are separate configs now LocalFileBasedAccessControlConfig and RestBasedAccessControlConfig. FileExists reverted for local file config